### DISCUSSION TITLE
Differences in results from pycocotools and this tools

### DISCUSSION THREAD

**Discussion by ghost** [rafaelpadilla/review_object_detection_metrics#149] [General]
Hi,

I've a coco ground truth file + a coco prediction file.
When I provide those files to the original implementation of pycocotools, I get these results : 
![image](https://github.com/user-attachments/assets/9b4e7cb5-cbfe-4683-bfd7-f0021bb0fe0c)

When I provide those files to your tool, I get these results : 
![image](https://github.com/user-attachments/assets/5b8fb160-35fc-496b-8ccc-0422fecee10d)

Why is there this slight difference (I understand that, regarding small/medium/large, you're using a different area)? 

Also, why your implementation differs from the official pycocotools ? https://github.com/cocodataset/cocoapi/blob/master/PythonAPI/pycocotools/cocoeval.py

Thanks for your help :)

**Comment by rafaelpadilla** [comment]
Hi,
Could you please provide the groundtruth and detection files used to produce these results?
